.github/workflows/go.yml

```diff
@@ -45,12 +45,14 @@ jobs:
     - name: Cross-build
       run: |
         # all GOOS/GOARCH combinations supported by `make local-release`
-        GOOS=linux GOARCH=386 go build ./...
-        GOOS=linux GOARCH=amd64 go build ./...
-        GOOS=linux GOARCH=arm go build ./...
-        GOOS=linux GOARCH=arm64 go build ./...
-        GOOS=darwin GOARCH=amd64 go build ./...
-        GOOS=darwin GOARCH=arm64 go build ./...
+        GOOS=linux GOARCH=386 go build ./cmd/cilium
+        GOOS=linux GOARCH=amd64 go build ./cmd/cilium
+        GOOS=linux GOARCH=arm go build ./cmd/cilium
+        GOOS=linux GOARCH=arm64 go build ./cmd/cilium
+        GOOS=darwin GOARCH=amd64 go build ./cmd/cilium
+        GOOS=darwin GOARCH=arm64 go build ./cmd/cilium
+        GOOS=windows GOARCH=amd64 go build ./cmd/cilium
+        GOOS=windows GOARCH=386 go build ./cmd/cilium
```

Comment:
NOTE: with Go 1.17, we could also build for `GOOS=windows GOARCH=arm64`

Reply:
Good point. Pushed an additional commit adding `windows/arm64` release binaries.